<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateClient</class>
 <widget class="QDialog" name="CreateClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Account</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page_3">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>40</y>
            <width>501</width>
            <height>271</height>
           </rect>
          </property>
          <property name="title">
           <string>Personal Details</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <property name="spacing">
              <number>35</number>
             </property>
             <item>
              <widget class="QLabel" name="label_name_3">
               <property name="text">
                <string>Last Name</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>lineEdit_lname</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_lname_3"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_27">
             <property name="spacing">
              <number>35</number>
             </property>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>First Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_fname_3"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_28">
             <property name="spacing">
              <number>56</number>
             </property>
             <item>
              <widget class="QLabel" name="label_address_3">
               <property name="text">
                <string>Address</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>lineEdit_address</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_address_3"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_dob_3">
               <property name="text">
                <string>Date Of Birth</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_dob</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_gen_3">
               <property name="text">
                <string>Gender</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>lineEdit_gender</cstring>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QRadioButton" name="radioButton_f">
                 <property name="text">
                  <string>Female</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_m">
                 <property name="text">
                  <string>Male</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_o">
                 <property name="text">
                  <string>Other</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_nationality_3">
               <property name="text">
                <string>Nationality</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_nationality</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_nationality_3"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>330</y>
            <width>146</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>58</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_prDetails">
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <widget class="QPushButton" name="pushButton_bnAcc_b">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>380</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_bnAcc_n">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>380</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>50</y>
            <width>481</width>
            <height>301</height>
           </rect>
          </property>
          <property name="title">
           <string>Bank Account</string>
          </property>
          <widget class="QLabel" name="label_acct_2">
           <property name="geometry">
            <rect>
             <x>13</x>
             <y>33</y>
             <width>92</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Account Type</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_acct</cstring>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget_2">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>80</y>
             <width>321</width>
             <height>161</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_25">
              <property name="spacing">
               <number>68</number>
              </property>
              <item>
               <widget class="QLabel" name="label_mob_2">
                <property name="text">
                 <string>Mobile No.</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_mob</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_mob_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_23">
              <property name="spacing">
               <number>85</number>
              </property>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Email ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_email_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_caste_2">
                <property name="text">
                 <string>User_Type</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_usrt</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox">
                <item>
                 <property name="text">
                  <string>Client</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Staff</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Account Type</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_acct">
                <item>
                 <property name="text">
                  <string>Savings</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Current</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Fixed Deposit</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>250</y>
             <width>221</width>
             <height>31</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Staff_ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>40</y>
            <width>451</width>
            <height>311</height>
           </rect>
          </property>
          <property name="title">
           <string>Security</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_26">
               <property name="spacing">
                <number>13</number>
               </property>
               <item>
                <widget class="QLabel" name="label_sq_2">
                 <property name="text">
                  <string>Security Question</string>
                 </property>
                 <property name="buddy">
                  <cstring>lineEdit_sq</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_sq_2"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Answer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_ans"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_usrname_crcl">
                 <property name="text">
                  <string>Username</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_username_crcl"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_password_crcl">
                 <property name="text">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_password_crcl"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton_sec_create">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>380</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_sec_b">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>380</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
